# Request
float64 traj_duration
float64 acc_duration
float64 end_position_x
float64 end_position_y
float64 end_position_z
string traj_type  
string s_type     
---
# Result - risultato finale dell'esecuzione
bool success
float64 final_error_norm
---
# Feedback - feedback continuo durante l'esecuzione
float64 error_x
float64 error_y
float64 error_z
float64 error_norm
float64 time_elapsed
